packages/block-library/src/button/edit.native.js

```diff
@@ -398,20 +408,21 @@ class ButtonEdit extends Component {
 			isButtonFocused,
 			placeholderTextWidth,
 		} = this.state;
+		const {
+			borderRadius: fallbackBorderRadius,
+			paddingTop: spacing,
+			borderWidth,
+		} = styles.defaultButton;
 
 		if ( parentWidth === 0 ) {
 			return null;
 		}
 
 		const borderRadiusValue =
-			borderRadius !== undefined
-				? borderRadius
-				: styles.button.borderRadius;
+			borderRadius !== undefined ? borderRadius : fallbackBorderRadius;
```

Comment:
```suggestion
			borderRadius || fallbackBorderRadius;
```

Reply:
It will not work properly when borderRadius is zero but refactored it to check if borderRadius is an integer!